config/routes/api.rb

```diff
@@ -22,6 +22,7 @@
         member do
           put :capture
           put :ship
+          put :update
```

Comment:
`update` is a default action. So it fits better in above:
```
      resources :orders, only: [:index, :show, :update] do
```

Reply:
Oops, I didn't know. Thanks. 🙏